docs/data/hubble/data-catalog/data-dictionary/claimable-balances.mdx

```diff
@@ -3,26 +3,40 @@ title: Claimable Balances
-| asset_type | The identifier for type of asset code, can be a alphanumeric with 4 characters, 12 characters or the native asset to the network, XLM. | string | credit_alphanum4 credit_alphanum12 native |  |  |  | Yes |  |
-| asset_code | The 4 or 12 character code representation of the asset on the network | string |  |  |  |  | No |  |
-| asset_issuer | The address of the account that created the asset | string |  |  |  |  | No |  |
-| asset_amount | The amount of the asset that can be claimed | float |  |  |  |  | Yes |  |
-| sponsor | The account address of the sponsor who is paying the reserves for this claimable balance | string |  |  |  |  | No | Sponsors of claimable balances are the creators of the balance. |
-| flags | Denotes the enabling and disabling of certain balance issuer privileges | integer | 0 - None, Default 1 - Auth Clawback Enabled |  |  |  | Yes | Flags are set by the claimable balance accounts for an asset. When user accounts claim a balance, the flags applied to the asset originate from this account |
-| last_modified_ledger | The ledger sequence number when the ledger entry (this unique signer for the account) was modified. Deletions do not count as a modification and will report the prior modification sequence number | integer |  |  | Yes | cluster | Yes |  |
-| ledger_entry_change | Code that describes the ledger entry change type that was applied to the ledger entry. | integer | 0 - Ledger Entry Created 1 - Ledger Entry Updated 2 - Ledger Entry Deleted 3 - Ledger Entry State (value of the entry) |  | Yes |  | Yes | Valid entry change types are 0, and 2 for ledger entries of type \`claimable_balances\`. Once created, a balance cannot be updated. |
-| deleted | Indicates whether the ledger entry (balance id) has been deleted or not. Once an entry is deleted, it cannot be recovered. | boolean |  |  |  |  | Yes |  |
-| batch_id | String representation of the run id for a given DAG in Airflow. Takes the form of "scheduled\_\_\<batch_end_date>-\<dag_alias>". Batch ids are unique to the batch and help with monitoring and rerun capabilities | string |  |  |  |  | Yes |  |
-| batch_run_date | The start date for the batch interval. When taken with the date in the batch_id, the date represents the interval of ledgers processed. The batch run date can be seen as a proxy of closed_at for a ledger. | datetime |  |  |  | MONTH partition | Yes | The table is partitioned on batch_run_date. It is recommended to always include the batch_run_date in the filter if possible to help reduce query cost. |
-| batch_insert_ts | The timestamp in UTC when a batch of records was inserted into the database. This field can help identify if a batch executed in real time or as part of a backfill | timestamp |  |  |  |  | Yes |  |
-| asset_id | Unique identifier for asset_code, asset_issuer | integer |  |  |  | cluster | No |  |
+# Claimable Balances
+
+|  |  |
+| --- | --- |
+| Primary Key(s) |  |
```

Comment:
yeah. I thought of keeping the table structure same and kept it with a BLANK. 
Will there be a need to create a unique key (like a generated id) for these tables instead of composite keys? If not, we can drop them completely I guess.

Reply:
Let's drop them completely. We have no plans to create a composite key right now